cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -160,57 +162,25 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
-        if (service === 'news') {
-          it('plays media when a user clicks play', () => {
-            cy.window().then(win => {
-              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
-              if (media && media.type === 'video') {
-                cy.get('div[class^="StyledVideoContainer"]')
-                  .within(() => {
-                    cy.get('button');
-                  })
-                  .click()
-                  .should('not.exist')
-                  .then(() => {
-                    cy.get('iframe[class^="StyledIframe"]').then($iframe => {
-                      cy.wrap($iframe.prop('contentWindow'), {
-                        // `timeout` only applies to the methods chained below.
-                        // `its()` benefits from this, and will wait up to 8s
-                        // for the mediaPlayer instance to become available.
-                        timeout: 20000,
-                      })
-                        .its('embeddedMedia.playerInstances.mediaPlayer')
-                        .invoke('currentTime')
-                        .should('be.gt', 0);
-                    });
-                  });
-              }
-            });
-          });
+        it('should render an iframe with a valid URL when a user clicks play', () => {
+          cy.window().then(win => {
+            const body = win.SIMORGH_DATA.pageData;
+            const media = getBlockData('video', body);
 
-          it('should have subtitles set to enabled by default', () => {
-            cy.window().then(win => {
-              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
-              if (media && media.type === 'video') {
-                cy.get(
-                  'div[class^="StyledVideoContainer"] iframe[class^="StyledIframe"]',
-                ).then($iframe => {
-                  cy.wrap($iframe.prop('contentWindow'), {
-                    // `timeout` only applies to the methods chained below.
-                    // `its()` benefits from this, and will wait up to 8s
-                    // for the mediaPlayer instance to become available.
-                    timeout: 20000,
-                  })
-                    .its(
-                      'embeddedMedia.playerInstances.mediaPlayer._settings.ui.subtitles.enabled',
-                    )
-                    .should('eq', true);
+            if (media && media.type === 'video') {
+              const { lang } = appConfig[service][variant];
+              const embedUrl = getVideoEmbedUrl(body, lang);
+              cy.testResponseCodeAndType(embedUrl, 200, 'text/html');
+              cy.get(
+                'div[class*="StyledVideoContainer"] button[class*="StyledPlayButton"]',
+              )
+                .click()
+                .then(() => {
+                  cy.get(`iframe[src="${embedUrl}"]`).should('be.visible');
```

Comment:
Is it also worth doing the embed URL check here too?

`cy.testResponseCodeAndType(embedUrl, 200, 'text/html');`

Reply:
Oops, you already do!